docker-bench-security.sh

```diff
@@ -19,6 +19,7 @@ dir_name=`dirname ${this_path}`    ## Dir where this file is
 myname=`basename ${this_path}`     ## file name of this script.
 logger="${myname}.log"
 
+export PATH=/bin:/sbin:/usr/bin
```

Comment:
Well, it removes the ability of running it on my laptop, which is obviously a big no-no ;)

How set are you on this exact directory list? Can we add `/usr/local/bin` to it?